### ISSUE TITLE
Include an option to immediately submit OpenAI requests instead of scheduling them

### ISSUE THREAD

**Issue by S0ulf3re** [nextcloud/assistant#10]
### Describe the feature you'd like to request

For requests sent to OpenAI's servers, the nextcloud assistant waits until cron is ran. This is understandable for larger scale servers. But isn't as useful on smaller servers where nowhere near as many requests will be made. The nextcloud assistant should have an administration option to immediately send a request instead of waiting around for cron to run.

### Describe the solution you'd like

There should be an option in the assistant settings to make it so that requests to OpenAI are immediately sent to their servers instead. As opposed to scheduling them.

### Describe alternatives you've considered

There is technically the ChatGPT extension that can be used from the smart picker. But often, I forget it's there, and I like having a button that I can click on to process editing.

**Comment by zydisney** [comment]
Agree! The significant delay of response is really a bed experience. Immediately reply in the pop-up window would be more efficient.

**Comment by theCalcaholic** [comment]
I don't think, this should be tied to external APIs - a better job scheduler that allows for immediate execution if there are no jobs in queue would make this feature much more useful.

**Comment by dhx** [comment]
This Issue might be relevant again for users after migrating to the upcoming Nextcloud 30 release considering the migration to task processing: https://github.com/nextcloud/assistant/issues/114
Running the `OC\TaskProcessing\SynchronousBackgroundJob` background job worker is recommended in the admin manual to avoid having to wait for the next cronjob run:
https://docs.nextcloud.com/server/latest/admin_manual/ai/app_assistant.html#improve-ai-processing-throughput

**Comment by janepie** [comment]
Unfortunately we do not have a solution to not need the background worker that wouldn't slow the whole server down immensely. It was discussed multiple times, we see no way around it.
Please have a look at https://docs.nextcloud.com/server/latest/admin_manual/ai/overview.html#ai-overview-improve-ai-task-pickup-speed to see how to set up the worker. I will leave this issue open so people can easier find and read it.

**Comment by TheSpaceGod** [comment]
Even if the background worker is necessary, can there be an easier way configured to enable it, especially for Nextcloud AIO where everything should be fairly controllable within the container or group of containers?
